cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1525,349 +1525,423 @@ function Start_Containerd {
   Log-Output "Starting containerd service"
   Start-Service containerd
 }
-
-# TODO(pjh): move the Stackdriver logging agent code below into a separate
+# TODO(pjh): move the logging agent code below into a separate
 # module; it was put here temporarily to avoid disrupting the file layout in
 # the K8s release machinery.
-$STACKDRIVER_VERSION = 'v1-11'
-$STACKDRIVER_ROOT = 'C:\Program Files (x86)\Stackdriver'
+$LOGGINGAGENT_VERSION = 'v1-11'
```

Comment:
This is unused right now - can you set it to `1.6.0` and use it down below?

Reply:
done. made the change.